src/components/Courses/Courses.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import CourseCard from './components/CourseCard/CourseCard';
+import SearchBar from './components/SearchBar/SearchBar';
+import Button from '../../common/Button/Button';
+
+export default function Courses({ course }) {
+	function showCreateCourse() {
+		document.getElementById('courses').style.display = 'none';
+		document.getElementById('createCoursePortal').style.display = 'block';
+	}
+
+	return (
+		<div id='courses'>
+			<div className='coursesBar'>
+				<SearchBar></SearchBar>
+				<Button buttonText='Add New Course' onClick={showCreateCourse}></Button>
+			</div>
+
+			{course?.map((course) => {
```

Comment:
1. Not clear usage: if you have array of courses then let's name it in corresponding way: `courses`
2. Usually, in React you should use the following statement to check whether any array item exists: `courses?.length ? courses .map(...) : null`